src/app/aliases/state/aliases.service.spec.ts

```diff
@@ -1,16 +1,17 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { provideHttpClientTesting } from '@angular/common/http/testing';
```

Comment:
`HttpClientTestingModule` is deprecated in Angular 18, the migration tool automatically updated the usages to `provideHttpClientTesting`